tests/nlu/base/test_utils.py

```diff
@@ -61,42 +76,30 @@ def test_ordered():
     assert ordered(target) == [("a", [1, 2, 3]), ("b", 1), ("c", "a")]
 
 
-@pytest.mark.parametrize(
-    ("model_dir", "expected"),
-    [
-        ("test_models/test_model_mitie/model_20170628-002704", True),
-        ("test_models/test_model_mitie_sklearn/model_20170628-002712", True),
-        ("test_models/test_model_spacy_sklearn/model_20170628-002705", True),
-        ("test_models/", False),
-        ("test_models/nonexistent_for_sure_123", False),
-    ],
-)
-def test_is_model_dir(model_dir, expected):
-    assert is_model_dir(model_dir) == expected
+def test_empty_is_model_dir(empty_model_dir):
+    assert is_model_dir(empty_model_dir)
```

Comment:
I feel like when using utils, we sometimes import the module and sometimes the functions. Is this something we can standardize?

Reply:
yes should import module and use `utils.is_model_dir ` - functions shouldn't be imported directly

Reply:
That's what I figured but wasn't sure since its allll over the code, haha. Do you want me to add a maintenance issue to go through and standardize it? Also do you have a suggestion for importing different utils modules, e.g. i've seen the combined one imported as `common_utils` and also `rasa_utils`

Reply:
yes it is but it is not important - we should just look for it in reviews as you did it and change it as we go. e.g. I'll change it in this file :+1: